recipes/baical-p7/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+import glob
+import os
+import shutil
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.28.0"
```

Comment:
```suggestion
required_conan_version = ">=1.29.0"
```

remove_files_by_mask is from 1.29.0